### PR TITLE
add other essential columns in user table

### PR DESCRIPTION
**Pull request by msm-faraj**
This pr aims to:
add other esential missing columns to the user table.

### CODE REVIEW

File: src/migrations/20240725114943-add-columns-on-user.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+"use strict";
+
+/** @type {import('sequelize-cli').Migration} */
```
Comment (pouyapourbaba) [inline src/migrations/20240725114943-add-columns-on-user.js:3]:
```suggestion
```

File: src/models/user.js [hunk 2]
```diff
@@ -14,9 +14,50 @@ module.exports = (sequelize, DataTypes) => {
   }
   User.init(
     {
-      firstName: DataTypes.STRING,
-      lastName: DataTypes.STRING,
-      email: DataTypes.STRING,
+      id: {
```
Comment (pouyapourbaba) [inline src/models/user.js:17]: These should be in a separate PR

Comment (msm-faraj) [reply src/models/user.js:17]: Is it possible to do anything this time or this is just for the next times?

Comment (pouyapourbaba) [reply src/models/user.js:17]: What?

File: src/migrations/20240725160713-install-uuid-ossp.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+/** @type {import('sequelize-cli').Migration} */
```
Comment (pouyapourbaba) [inline src/migrations/20240725160713-install-uuid-ossp.js:1]:
```suggestion
```

**Comment by msm-faraj** [comment]
we decided to clean up everything about migrtation and database
